tests/ensure_provider_tests.py

```diff
@@ -86,7 +81,7 @@
     "playbooks/tests_provider.yml": {
         MINIMUM_VERSION: "'1.20.0'",
         "comment": "# NetworKmanager 1.20.0 added support for forgetting profiles",
-        EXTRA_RUN_CONDITION: "ansible_distribution != 'RedHat' or\n      ansible_distr\
+        EXTRA_RUN_CONDITION: "not ansible_distribution in ['RedHat', 'CentOS'] or\n      ansible_distr\
```

Comment:
Is this change necessary? `playbooks/tests_802_1x_updated.yml` and `playbooks/tests_802_1x.yml` also have the the same EXTRA_RUN_CONDITION, but the condition is not changed.

Reply:
It is not related to this PR and I have removed it.  However, this test fails on centos-9 and should be fixed, probably in the manner I have above (basically, anything that should be done on `RedHat` should also be done on `CentOS`), in a subsequent PR.